builder.md

```diff
@@ -99,6 +100,7 @@ A builder MUST provide the following variables to template files:
 - `scheme-name` - obtained from the `scheme` key of the scheme file
 - `scheme-author` - obtained from the `author` key of the scheme file
 - `scheme-description` - obtained from the `description` key of the scheme file (fallback value: `scheme-name`)
+- `is_{{variant}}_variant` - dynamic value obtained from the `variant` key of the scheme file. e.g. `variant: "light"` provides `is_light_variant` with a value of `true`. (fallback value: `is_dark_variant` with a value of `true`). Officially supported values are `light` and `dark`, but any string could theoretically be used.
```

Comment:
`scheme-variant` should probably also be exported.

Reply:
As an extra line? (I've updated it)